jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -13,7 +13,7 @@
     <servlet-class>org.eclipse.jetty.ee10.proxy.ProxyServlet$Transparent</servlet-class>
     <init-param>
       <param-name>proxyTo</param-name>
-      <param-value>https://eclipse.dev/jetty/javadoc/jetty-12/index.html?overview-summary.html</param-value>
+      <param-value>https://javadoc.jetty.org/jetty-12/index.html</param-value>
```

Comment:
Wrong URL

Reply:
sure?

Reply:
The lack of query section threw me on this one.
But it's correct.